index.html

```diff
@@ -1,19 +1,150 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Karma</title>
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
 </head>
+
 <body>
+    <header>
+        <nav class="navigation ">
+
+
+            <ul class="flexboxrow navul">
+                <li><svg class="logo-svg" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 144 216">
+                        <path fill="#F15B2A"
+                            d="M126.1 212.9c4.1 4.1 10.7 4.1 14.8 0 4.1-4.1 4.1-10.7 0-14.8l-28.6-28.6 18.1-18.1c8.4-8.4 13.6-20 13.6-32.9C144 92.8 123.2 72 97.5 72S51 92.8 51 118.5c0 12.8 5.2 24.5 13.6 32.9l18.1 18.1-18.1 18.1c-4.6 4.6-11 7.5-18.1 7.5-14.1 0-25.6-11.5-25.6-25.6v-159C20.9 4.7 16.2 0 10.5 0 4.7 0 0 4.7 0 10.5v159C0 195.2 20.8 216 46.5 216c12.8 0 24.5-5.2 32.9-13.6l18.1-18.1 28.6 28.6zm-46.7-76.3c-4.6-4.6-7.5-11-7.5-18.1 0-14.1 11.5-25.6 25.6-25.6 14.1 0 25.6 11.5 25.6 25.6 0 7.1-2.9 13.5-7.5 18.1l-18.1 18.1-18.1-18.1z" />
+                    </svg></li>
+                <li class="li-space"></li>
+
+                <li ><strong><a class="nav-li" href="#">Meet Karma</a></strong></li>
+                <li ><a class="nav-li"href="#">How is work</a></li>
+                <li ><a class="nav-li"href="#">Store</a></li>
+                <li ><a class="nav-li"href="#">Blog</a></li>
+                <li ><a class="nav-li"href="#">Help</a></li>
+                <li ><a class="nav-li"href="#">Login</a></li>
+
+            </ul>
+
+        </nav>
+    </header>
+    <main>
+        <div class="main-picture">
+            <h1>
+                Introducing Karma
+            </h1>
+            <h2>
+                 Bring WiFi with you, everywhere you go.
+            </h2>
+            <button class="button-red-text-white">Learn more</button>
+        </div> 
+         <section class="bottom-main">
+            <div class="text-bottom-main">
+                Everyone needs a little Karma.
+            </div>
+            <div class="three-images">
+                <div>
```

Comment:
Again, consider [semantic tags](https://www.w3schools.com/html/html5_semantic_elements.asp) - fould `figure` work here better?

The more diversity you have in the markup, the more understandable your styles are. Instead of very generic divs you can express the intent much more clearly.